src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.courses.domain;
+
+public class Image {
+    private String fileName;
+    private String contentType;
+    private long sizeInBytes;
+    private int width;
+    private int height;
+    private static final int IMAGE_SIZE_1MB = 1024 * 1024;
```

Comment:
```suggestion
    private static final int MAX_IMAGE_SIZE = 1024 * 1024;
```
1024 * 1024 값이 1MB라는 것은 중요하지 않다고 생각해요 
이 값이 무슨 의미를 가지는지 나타내는 것이 더 좋다고 느껴집니다!